src/pages/talks/_talk.js

```diff
@@ -0,0 +1,272 @@
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Link from 'gatsby-link';
+
+import imagePatternDots from '../../assets/images/patterns/dots.png';
+import {
+  Header,
+  Drops,
+  Footer,
+  TechnologyLogo,
+  Button
+} from '../../components';
+import { Contact, SignUp } from '../../containers';
+import technologies from '../../data/technologies.json';
+import talks from '../../data/talks.json';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    minHeight: '100vh',
+    flexDirection: 'column'
+  },
+
+  // MAIN
+  main: {
+    flex: '1',
+    '& h1': {
+      margin: 0
+    }
+  },
+  mainContainer: {
+    padding: [20, 0, 0],
+    '& .row': {
+      margin: 0
+    },
+    '& .col-xs-12': {
+      padding: 0
+    }
+  },
+  videoContainer: {
+    marginBottom: 20
+  },
+  videoFrame: {
+    width: '100%',
+    height: 200
+  },
+  leftContent: {
+    maxWidth: '100%',
+    marginTop: 20,
+    padding: [0, 20]
+  },
+  logoContainer: {
+    display: 'flex',
+    alignItems: 'stretch'
+  },
+  logo: {
+    width: '100%',
+    padding: [25, 0],
+    margin: [8, 0]
+  },
+  subtitle: {
+    fontStyle: 'italic',
+    fontFamily: theme.typography.secondary,
+    color: theme.palette.text.base,
+    marginBottom: 30
+  },
+  backButton: {
+    marginTop: 20,
+    display: 'none'
+  },
+  topButton: {
+    margin: [0, 0, 20, 20],
+    display: 'flex'
+  },
+
+  // RIGHT CONTENT
+  rightTitle: {
+    marginTop: 10,
+    fontSize: 16,
+    marginBottom: 10,
+    backgroundColor: '#EAEAEA',
+    padding: [10, 20],
+    '& span': {
+      display: 'none'
+    }
+  },
+  button: {
+    width: '100%',
+    marginBottom: 20
+  },
+
+  // FOOTER
+  footer: {
+    backgroundColor: theme.palette.inverted.base,
+    backgroundImage: `url(${imagePatternDots})`
+  },
+  footerDrops: {
+    display: 'none'
+  },
+
+  // small +
+  '@media screen and (min-width: 48em)': {
+    // MAIN
+    mainContainer: {
+      padding: [40, 20, 20],
+      '& .row': {
+        margin: [0, -8]
+      },
+      '& .col-xs-12': {
+        padding: [0, 8]
+      }
+    },
+    videoFrame: {
+      height: 530
+    },
+    backButton: {
+      display: 'block'
+    },
+    topButton: {
+      display: 'none'
+    },
+    leftContent: {
+      maxWidth: 550,
+      padding: 0
+    },
+
+    // RIGHT CONTENT
+    rightTitle: {
+      marginTop: 40,
+      backgroundColor: 'transparent',
+      padding: 0,
+      '& span': {
+        display: 'inline-block'
+      }
+    },
+
+    button: {
+      display: 'none'
+    },
+    logo: {
+      border: [1, 'solid', '#EAEAEA']
+    },
+
+    // FOOTER
+    footerDrops: {
+      display: 'block'
+    }
+  }
+});
+
+const Talk = ({ classes, children }) => {
+  const {
+    id,
+    videoUrl,
+    title,
+    subtitle,
+    content,
+    specialties,
+    nextTalk
+  } = children;
+  const nexTalk = talks.find(
+    (item, index) => item.id === id && !nexTalk && talks[index + 1]
```

Comment:
I don't know how it worked, it should have failed. The variable should have a different name and you should check first if it is not already provided. In this case, you are looking the whole array even if you don't need to because it was provided.